org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/Messages.java

```diff
@@ -76,6 +78,11 @@ private Messages() {
   public static String deleteServerDialogBoundProject;
   public static String deleteServerTask;
 
+  public static String unbindProjectDialogTitle;
+  public static String unbindProjectDialogMessage;
+  public static String unbindProjectDialogMessageMany;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)